### ISSUE TITLE
How can I get the camera pose information after the reconstruction?

### ISSUE THREAD

**Issue by HDUExia** [colmap/colmap#2284]
The images that I want to reconstruct has the GPS information.

I want to know the coordinate(camera pose information) after reconstruction.

With that I can easily mark every cloud point with a global coordinate in longitude and latitude.

Is there any way to get this information from colmap?

Looking forward to your reply.

**Comment by tsattler** [comment]
Yes, it is stored in the text respectively binary output files. Please see the documentation / tutorial for details.

**Comment by HDUExia** [comment]
Thank for your reply, After reading the issue #2222, I modify the python script (scripts/python/read_write_model.py) in cpp, and it work.successfully read the camera pose.

**Comment by ahojnnes** [comment]
Closing: camera poses stored in output files; user confirmed.
